### DISCUSSION TITLE
xray分流后，ipv6检测网站打不开

### DISCUSSION THREAD

**Discussion by yishion819** [Openwrt-Passwall/openwrt-passwall2#662] [Q&A]
直连规则添加了v6检测网站后，过一段时间后，test-ipv6.com死活打不开，貌似直连规则会失效？请大佬赐教

**Comment by ghost** [comment]
如果您使用的不是x86設備，且設備效能較低（例如arm的無線路由器）則很有可能會出現這個問題，根本原因是RAM空間不足，其空間不能允許指定量ipset規則加載到RAM中所致。
如果是以上情形，請考慮更換設備，以徹底解決這個問題。
如果不是上述原因，且使用的是x86設備，請考慮是否是因爲passwall所需依賴（如dnsmasq-full* kmod-ipt* iptables* ip6tables* kmod-ip6*） 在固件編譯時沒有選全或者內核版本太舊了所致的。

**Reply by yishion819** [reply]
谢谢大佬指点，是x86，内核5.15.132，我查了依赖，你说的我只有dnsmasq-full安装了，但后面几个依赖确实没有，我搜软件包看到貌似有很多，请问是后台直接安装你说的这几个依赖么？
